### DISCUSSION TITLE
Add brief instructions for running Mullvad VPN CLI from docker or Podman

### DISCUSSION THREAD

**Discussion by caewok** [mullvad/mullvadvpn-app#9079] [Feature requests & Ideas]
### I have checked if others have suggested this already

- [X] I have checked this issue tracker to see if others have reported similar issues.

### Feature description

I have been trying to get Mullvad VPN CLI working from docker. It is non-trivial, because mullvad relies on `systemd` and Docker containers usually do not run `systemd`. From what I can tell, it looks like podman should be able to run `systemd` containers, but I have had no luck making that work. 

From reading the issues in the Git here, it seems like others had similar problems but that a few commenters had successfully set up a docker container with the mullvad cli. So I thought it might be helpful to the community if someone who has gotten this to work could provide some instructions in the README or in wiki.

Related issues: #5431, #5408, #4894.

### Alternative solutions

I tried https://github.com/gvkhna/docker-mullvadvpn/ without success. I also tried installing mullvad cli from a podman container, without success.

### Type of feature

- [ ] Better privacy/anonymity
- [ ] Better at circumventing censorship
- [ ] Easier to use
- [X] Other

### Operating System

- [ ] Android
- [ ] iOS
- [ ] Windows
- [ ] macOS
- [X] Linux

**Comment by the-me-0** [comment]
Hi @caewok !
I actually did manage to make the whole thing to run (and on arm64 on top of that). Based on [@gvkhna](https://github.com/gvkhna/docker-mullvadvpn/)'s repository, 10 hours of troubleshooting, and with the help of the issue you posted in his repository.
Let me just create a repository on my side and I will link it right below in just a moment.

**Comment by the-me-0** [comment]
Can be accessed here : [https://github.com/the-me-0/docker-mullvadvpn](https://github.com/the-me-0/docker-mullvadvpn)

**Comment by hemidark** [comment]
The addition of split tunneling also seems to block many/most container-oriented use cases now, per #3737
